### ISSUE TITLE
Allow setting custom ollama API endpoint location

### ISSUE THREAD

**Issue by Bisaloo** [ropensci-review-tools/pkgmatch#83]
(I may be misunderstanding something so please let me know / close this issue if that's the case.)

From what I understand, pkgmatch currently only supports using ollama on localhost (https://github.com/ropensci-review-tools/pkgmatch/blob/3a26604450eceee7770c7a6f212c97101ee96aa3/R/ollama.R#L131).

But there might be cases where a "local LM" is still not hosted on 127.0.0.1. I encountered this situation when trying to use pkgmatch with docker compose, where each container gets its own IP on their internal network.

Similarly, one may host the LM on another device on their internal network.

Alternatively, it might be set up to listen to a different port than the default.

Would it be possible to customize the API endpoint location through an environment variable, with the default being the current setting & ollama default?

**Comment by mpadge** [comment]
Yeah good idea, thanks @Bisaloo, that would be easy to incorporate.

**Comment by Bisaloo** [comment]
Would it make sense to also set it as an environment var so it can be set conveniently outside of R, e.g. in docker config, or CI?

There was some interesting discussion about `options()` vs env var on mastodon some time ago: https://fosstodon.org/@coolbutuseless/112119205252695034

**Comment by mpadge** [comment]
Yeah, right, using `Sys.getenv()` during `.onLoad()` would be a neat idea. I'll re-open and do that, but feel free to PR if you like in the meantime ...
